Firestore/Source/Core/FSTFirestoreClient.mm

```diff
@@ -129,6 +130,7 @@ @implementation FSTFirestoreClient {
   std::chrono::milliseconds _initialGcDelay;
   std::chrono::milliseconds _regularGcDelay;
   BOOL _gcHasRun;
+  std::atomic<bool> _isShutdown;
```

Comment:
Please change all `BOOL` to `bool`, `YES` to `true`, `NO` to `false` in this file.